### PR TITLE
Bug fix for: Get paid navigation crashing from transfer fragment.

### PR DESCRIPTION
**Pull request by androidmaven**
- Cause was that AbstractFormFragment was trying to make use of the same observer from different lifecycles.

- Solution was to move those observers to their individual lifecycle i.e fragment.

### CODE REVIEW

**Comment by davkutalek** [comment]
@androidmaven this explanation doesn't quite make sense: Each of those fragments is inheriting from AbstractFormFragment and their lifecycles are totally separate even tho they share the same code base.

The cause appears to be something weird in the compiler that causes it to treat the observer as static if it appears to be the same thing. See this for a sort of expanation https://stackoverflow.com/questions/47025233/android-lifecycle-library-cannot-add-the-same-observer-with-different-lifecycle maybe @alexosugo can shed more light.

I was able to fix it the way that post says by changing the logging calls:
`Timber.e("Got new actions ${this.javaClass.simpleName}: %s", it?.size)`
I think this is preferable since otherwise the abstract class is delegating required set-up to its inheritors.

**Review by davkutalek — CHANGES_REQUESTED** [review]
See above comment

File: app/src/main/java/com/hover/stax/RoutingActivity.kt [hunk 1]
```diff
@@ -115,7 +115,7 @@ class RoutingActivity : AppCompatActivity(), BiometricChecker.AuthListener, Push
 
     private fun initHover() {
         Hover.initialize(this)
-        Hover.setBranding(getString(R.string.app_name), R.mipmap.stax, R.drawable.ic_stax, this)
+        Hover.setBranding(getString(R.string.app_name), R.mipmap.stax,  this)
```
[the remark this answers is not in the document]
Comment (androidmaven) [reply app/src/main/java/com/hover/stax/RoutingActivity.kt]: Done, thanks.

**Comment by androidmaven** [comment]
> 

Thanks for the correction @davkutalek.

I did check out the StackOverflow link earlier while attempting to fix the bug, but over-looked particularly ${this.javaClass.simpleName}.

**Review by davkutalek — APPROVED** [review]
LGTM

**Pull request merged**